### ISSUE TITLE
Extension doesn't work

### ISSUE THREAD

**Issue by insanetesterftw** [bartosz-antosik/vscode-spellright#6]
Using Windows 7 64bit, Visual Studio Code latest stable version.

Extension is installed.

Tested on a text file (plain). Doesn't highlight any of the mistakes that exist.

Executing any of the "SpellRight" commands returns "command not found"

Other installed extensions: asciidoc; beautify; html preview; markdownlint; rewrap; write good linter.

I'm frustrated that I can't provide better feedback because I can't find any reason from my part for the failure of the extension to work.

Thank you.

**Comment by bartosz-antosik** [comment]
Could you tell me what is displayed in the language switch (a quick list at editor's toolbar) when you click language name in the status bar? Is the language selector in the status bar displayed at all?

**Comment by bartosz-antosik** [comment]
I did some tests and it seems that on freshly installed W7+VSCode+SpellRight (as the only extension) it does not show symptoms you describe ("command not found"). It does not spell and show empty dictionary list but this is another matter - on Windows 8+ extension uses Windows Spell Checking API whereas on W7 it uses hunspell speller which does not have any dictionaries installed.

![screenshot](https://cloud.githubusercontent.com/assets/20492548/25017853/7e76926c-2085-11e7-855c-fb52b0e48485.png)

So I need some more clues to do something about it.

**Comment by kwzm** [comment]
me too

**Comment by farfromrefug** [comment]
Same thing here 
Here is what i see in the dev console
```
Activating extension `ban.spellright` failed: dlopen(/Volumes/data/mguillon/.vscode-insiders/extensions/ban.spellright-1.0.6/node_modules/spellchecker/build/Release/spellchecker.node, 1): no suitable image found.  Did find:
	/Volumes/data/mguillon/.vscode-insiders/extensions/ban.spellright-1.0.6/node_modules/spellchecker/build/Release/spellchecker.node: unknown file type, first eight bytes: 0x4D 0x5A 0x90 0x00 0x03 0x00 0x00 0x00
	/Volumes/data/mguillon/.vscode-insiders/extensions/ban.spellright-1.0.6/node_modules/spellchecker/build/Release/spellchecker.node: unknown file type, first eight bytes: 0x4D 0x5A 0x90 0x00 0x03 0x00 0x00 0x00.
t.$localShowMessage @ mainThreadExtensionService.ts:111
messageService.ts:126command 'spellright.createUpdateSettings' not found
```

**Comment by bartosz-antosik** [comment]
@farfromrefug Could you please tell me what OS is this?

Paths do not look like Windows 64-bit.

**Comment by bartosz-antosik** [comment]
I have verified that it works in Windows 7 64-bit. It was missing information on how to add dictionaries (now in README.md) and the language switch did not support switching languages in Windows 7. The later has been corrected in version 1.0.7.

![w7-screenshot-1](https://cloud.githubusercontent.com/assets/20492548/25129649/3401737e-243f-11e7-915c-93364fcad72e.png)

**Comment by farfromrefug** [comment]
@insanetesterftw i am on macOs. I now see in the readme that it just won't work ...

**Comment by lex111** [comment]
On Ubuntu this extension doesn't work :(

**Comment by bartosz-antosik** [comment]
Yes, it does not work on UBuntu neither it does on macOS. It is clearly written in first lines of README that at the moment it works only on 64 bit Windows. The reason for this is explained with a lot of details [here](https://github.com/Microsoft/vscode/issues/20266).

TL;DR; VSCode team should build include one module in the builds for all platforms.
